packages/core/admin/admin/src/core/apis/CustomFields.js

```diff
@@ -0,0 +1,80 @@
+import invariant from 'invariant';
+
+const ALLOWED_TYPES = [
```

Comment:
Isn't it the same as the CTB? Can we have an API or something so we can sync the allowed types in the ctb with the ones in the CM ?

Reply:
I think think we allow less fields, relation is not allowed for example

Reply:
Good point I forgot about that!